view/src/components/produce.js

```diff
@@ -1,674 +1,710 @@
-			.then(() => {
-				window.location.reload();
-			})
-			.catch((err) => {
-				console.log(err);
-			});
-	}
+  deleteTodoHandler(data) {
+    authMiddleWare(this.props.history);
+    const authToken = localStorage.getItem("AuthToken");
+    axios.defaults.headers.common = { Authorization: `${authToken}` };
+    let produceId = data.produce.produceId;
+    axios
+      .delete(`produce/${produceId}`)
+      .then(() => {
+        window.location.reload();
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  }
 
-	handleEditClickOpen(data) {
-		this.setState({
-			name: data.produce.name,
-			produceId: data.produce.produceId,
-			shippingPresetTemperature: data.produce.shippingPresetTemperature,
-			shippingMaintenanceTemperatureLow: data.produce.shippingMaintenanceTemperatureLow,
-			shippingMaintenanceTemperatureHigh: data.produce.shippingMaintenanceTemperatureHigh,
-			amountMoved: data.produce.amountMoved,
-			price: data.produce.price,
-			pricePaid: data.produce.pricePaid,
-			buttonType: 'Edit',
-			open: true
-		});
-	}
+  handleEditClickOpen(data) {
+    this.setState({
+      name: data.produce.name,
+      produceId: data.produce.produceId,
+      shippingPresetTemperature: data.produce.shippingPresetTemperature,
+      shippingMaintenanceTemperatureLow:
+        data.produce.shippingMaintenanceTemperatureLow,
+      shippingMaintenanceTemperatureHigh:
+        data.produce.shippingMaintenanceTemperatureHigh,
+      amountMoved: data.produce.amountMoved,
+      price: data.produce.price,
+      pricePaid: data.produce.pricePaid,
+      buttonType: "Edit",
+      open: true,
+    });
+  }
 
-	handleViewOpen(data) {
-		this.setState({
-			name: data.produce.name,
-			shippingPresetTemperature: data.produce.shippingPresetTemperature,
-			shippingMaintenanceTemperatureLow: data.produce.shippingMaintenanceTemperatureLow,
-			shippingMaintenanceTemperatureHigh: data.produce.shippingMaintenanceTemperatureHigh,
-			amountMoved: data.produce.amountMoved,
-			price: data.produce.price,
-			pricePaid: data.produce.pricePaid,
-			viewOpen: true
-		});
-	}
+  handleViewOpen(data) {
+    this.setState({
+      name: data.produce.name,
+      shippingPresetTemperature: data.produce.shippingPresetTemperature,
+      shippingMaintenanceTemperatureLow:
+        data.produce.shippingMaintenanceTemperatureLow,
+      shippingMaintenanceTemperatureHigh:
+        data.produce.shippingMaintenanceTemperatureHigh,
+      amountMoved: data.produce.amountMoved,
+      price: data.produce.price,
+      pricePaid: data.produce.pricePaid,
+      viewOpen: true,
+    });
+  }
 
-	handleSearch = event => {
-		const { value } = event.target;
-		this.setState({ value });
-	  };
+  // This function updates the string that the produce array will be searched for.
+  // This string value can contain any produce field (name, weight, interrnal shipment numbers) and will still return
+  // 	the appropriate filtered results.
```

Comment:
Suggest shortening the first line here so it'll fit better on my screen :P

The google style guide for html recommends 100 character line length (but i believe 80 for js, so...take your pick). I have my IDE draw lines at 80 & 100 characters so I can see when I'm going over the limit.